asistente_ladm_col/gui/qgis_model_baker/dlg_import_data.py

```diff
@@ -439,53 +421,30 @@ def restore_configuration(self):
         if self.use_local_models:
             self.custom_model_directories = self.app.settings.custom_model_dirs
 
-    def update_configuration(self):
+    def update_configuration(self, ili2db: Ili2DB):
         """
         Get the configuration that is updated with the user configuration changes on the dialog.
         :return: Configuration
         """
-        db_factory = self._dbs_supported.get_db_factory(self.db.engine)
+        disable_validation = \
+            not QSettings().value('Asistente-LADM-COL/models/validate_data_importing_exporting', True, bool)
 
-        configuration = ImportDataConfiguration()
-        db_factory.set_ili2db_configuration_params(self.db.dict_conn_params, configuration)
+        configuration = ili2db.get_import_data_configuration(self.db, self.xtf_file_line_edit.text().strip(),
+                                                             disable_validation=disable_validation)
 
-        configuration.xtffile = self.xtf_file_line_edit.text().strip()
         configuration.delete_data = False
 
         configuration.srs_auth = QSettings().value('Asistente-LADM-COL/QgisModelBaker/srs_auth', DEFAULT_SRS_AUTH, str)
         configuration.srs_code = QSettings().value('Asistente-LADM-COL/QgisModelBaker/srs_code', int(DEFAULT_SRS_CODE), int)
-        configuration.inheritance = ILI2DBNames.DEFAULT_INHERITANCE
+
         configuration.create_basket_col = ILI2DBNames.CREATE_BASKET_COL
         configuration.create_import_tid = ILI2DBNames.CREATE_IMPORT_TID
         configuration.stroke_arcs = ILI2DBNames.STROKE_ARCS
```

Comment:
These are taking default values, which are anyways set inside Ili2Db's methods, so I guess we could remove them from here.